test/testenv.py

```diff
@@ -25,6 +25,7 @@ def get_testcases(path):
             yield os.path.join(path, f)
 
 options_schema = strictyaml.Map({
+    'domain': strictyaml.Str(),
```

Comment:
Domain could be an Enum to limit the values: https://hitchdev.com/strictyaml/using/alpha/scalar/enum/

I guess it's cleaner to require adding the domain instead of making it optional with 'c' as default?

Reply:
Nice idea about the enum. Otherwise, yes, I'd rather make it explicit, especially as ultimately there should be no reason to presume or guess that one or the other is the default other than historic reasons.